#water: [0.1, 0.2, 0.3]
#toilet_paper: [0.2, 0.2, 0.2]
#medicine: [0.3, 0.3, 0.3]
#bed_room: [1.0, 1.0, 1.0]
#toilet: [2.0, 2.0, 2.0]
#kitchen: [3.0, 3.0, 3.0]
#front_door: [4.0, 4.0, 4.0]
#living_room: [5.0, 5.0, 5.0]
water: 1
toilet_paper: 2
medicine: 3
#bed_room: 10
#toilet: 20
#kitchen: 30
#front_door: 40
#living_room: 50